DTMD.xsd

```diff
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning" xmlns:dtmd="http://apps.pmsf.net/STMD/DecisionTaskMetaData" xmlns:stc="http://apps.pmsf.net/SSPTraceability/SSPTraceabilityCommon" xmlns:ssc="http://ssp-standard.org/SSP1/SystemStructureCommon" targetNamespace="http://apps.pmsf.net/STMD/DecisionTaskMetaData" elementFormDefault="qualified" vc:minVersion="1.1">
+	<xs:import namespace="http://ssp-standard.org/SSP1/SystemStructureCommon" schemaLocation="ssp-standard/schema/SystemStructureCommon.xsd"/>
+	<xs:import namespace="http://apps.pmsf.net/SSPTraceability/SSPTraceabilityCommon" schemaLocation="STC.xsd"/>
+	<xs:element name="DecisionTaskMetaData">
+		<xs:complexType>
+			<xs:sequence>
+				<xs:element name="GeneralInformation" minOccurs="0" type="stc:GeneralInformationType"/>
+				<xs:element name="AnalysisPhase" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation xml:lang="en">
+							This element specifies the Analysis Phase of the overall Decision Task.
+						</xs:documentation>
+					</xs:annotation>
+					<xs:complexType>
+						<xs:sequence>
+							<xs:element name="AnalyzeEngineeringTask" type="stc:StepType" minOccurs="0"/>
+							<xs:element name="VerifyAnalysis" type="stc:StepType" minOccurs="0"/>
+							<xs:group ref="stc:GPhaseCommon"/>
+						</xs:sequence>
+						<xs:attributeGroup ref="ssc:ABaseElement"/>
+					</xs:complexType>
+				</xs:element>
+				<xs:element name="DefinitionPhase" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation xml:lang="en">
+							This element specifies the Definition Phase of the overall Decision Task,
+							where the individual sub-tasks to be performed are defined.
+						</xs:documentation>
+					</xs:annotation>
+					<xs:complexType>
+						<xs:sequence>
+							<xs:element name="DefineTasks" type="stc:StepType" minOccurs="0"/>
+							<xs:element name="DefineResultQuality" type="stc:StepType" minOccurs="0"/>
+							<xs:element name="VerifyTasks" type="stc:StepType" minOccurs="0"/>
+							<xs:group ref="stc:GPhaseCommon"/>
+						</xs:sequence>
+						<xs:attributeGroup ref="ssc:ABaseElement"/>
+					</xs:complexType>
+				</xs:element>
+				<xs:element name="ExecutionPhase" minOccurs="0">
+					<xs:annotation>
+						<xs:documentation xml:lang="en">
+							This element specifies the Performance Phase of the overall Decision Task.
+						</xs:documentation>
+					</xs:annotation>
+					<xs:complexType>
+						<xs:sequence>
+							<xs:element name="PeformTasks" type="stc:StepType" minOccurs="0"/>
```

Comment:
Name attribute should be "Pe**r**formTasks"

Reply:
Good catch...